### DISCUSSION TITLE
Lab Abnormality KRI

### DISCUSSION THREAD

**Discussion by jwildfire** [Gilead-Public/gsm#540] [General]
@gwu05 I'm working on an issue detailing the  lab abnormality assessment, and I'm thinking the KRI should be '% of abnormal labs' instead of 'Abnormal Labs/Month'. Seems like it would make it a bit more stable across studies since the visit scheduled and lab volume is likely to be highly variable. 

What do you think?

**Comment by gwu05** [comment]
As discussed, I think either is okay to include, where the difference is going to be driven by the denominator of # of total labs or # of total time on treatment. Essentially, if a site has a larger number of abnormal labs, they should be pinged on either KRI, as in this particular case the # of total labs and the # of total time on treatment are usually correlated as both are trying to represent and account for the amount of study participation.

Of course, there are different potential scenerios in which one may do better than the other:

- For example, if there are a large number of repeat labs across visits that result in abnormal findings ( e.g. like a kidney safety issue ), then there will be a unusual spike in Abnormal Labs/Month, which would likely be better account for using % of abnormal labs.

- For example, if there are a small number of labs, then % of abnormal labs may have more issues as the % can vary substantially by the number of total labs run (e.g. if only 5 labs were run then 1 abnormal lab is going from 0% => 20%), while Abnormal Labs / month would likely be able to better account for this since time in months is not discrete like total labs, even for shorter study times.

- The choice typically is also interpretation-related (is the question about event in a time interval or # of events out of total measurements), but as explained above our situation, both KRIs are trying to get at the same thing.
